genlib/bspnode.cpp

```diff
@@ -0,0 +1,134 @@
+#include "bspnode.h"
```

Comment:
No copyright header? I assume this is just a refactor moving the code to its own file? In which case copyright should probably be Alasdair...